rust_src/src/print.rs

```diff
@@ -0,0 +1,46 @@
+use remacs_macros::lisp_fn;
```

Comment:
Minor style nit.

We start each file with a comment details what it contains. You can re-use the C code's comment here:

`//! Lisp object printing and output streams.`